modules/controlled_access_terms_defaults/config/install/field.storage.taxonomy_term.field_external_authority_link.yml

```diff
@@ -1,17 +0,0 @@
-langcode: en
-status: true
-dependencies:
-  module:
-    - taxonomy
-id: taxonomy_term.field_external_authority_link
-field_name: field_external_authority_link
-entity_type: taxonomy_term
-type: authority_link
-settings: {  }
-module: core
-locked: false
-cardinality: 1
```

Comment:
@seth-shaw-unlv  - note that the biggest change here between the field_external_authority_link and field_authority_link is the cardinality value changes from 1 to -1.  Probably not a big deal because you could put a link to MARC as well as DCMI. Not sure if people would though.

Reply:
I prefer consistency and can also imagine using multiple authority links; so changing it to '-1' seems best.